resources/build/tests/build-utils-traps.test.sh

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env bash
+
+set -eu
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/../build-utils.sh"
+# END STANDARD BUILD SCRIPT INCLUDE
+
+. "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
+
+##############
+
+CHECK="${COLOR_GREEN}\xE2\x9C\x94${COLOR_RESET}" # ✔
+CROSS="${COLOR_RED}\xE2\x9D\x8C${COLOR_RESET}" # ❌
```

Comment:
```suggestion
CHECK="${COLOR_GREEN}✔${COLOR_RESET}"
CROSS="${COLOR_RED}❌${COLOR_RESET}"
```

Any reason we can't just do this?

Reply:
It does look fine in Windows/MINGW64; guess I should've tried it out first.  Interestingly, the references I found when I went looking for how to do this all just used the coded form... maybe it would've been a problem in the past?